pkg/provider/aci.go

```diff
@@ -472,8 +474,11 @@ func (p *ACIProvider) deleteContainerGroup(ctx context.Context, podNS, podName s
 			false,
 		)
 
-		if updateErr != nil && !errdefs.IsNotFound(updateErr) {
-			log.G(ctx).WithError(updateErr).Errorf("failed to update termination status for cg %v", cgName)
```

Comment:
Why do you change ths? updateErr is not returned by SDK call

Reply:
Good catch. Fixed.